docs/source/python.rst

```diff
@@ -0,0 +1,80 @@
+ome_zarr python library
+=======================
+
+writing OME-NGFF images
```

Comment:
```suggestion
Writing OME-NGFF images
```

Reply:
More important comment though:

The bullet point with a link to this doc is tallking about writing ``filesets``. Should this title and text also rather talk about filesets ?